src/nxt_conf_validation.c

```diff
@@ -1465,6 +1515,73 @@ nxt_conf_validate(nxt_conf_validation_t *vldt)
     "a number, a string, an array, or an object"
 
 
+
+#if (NXT_HAVE_OTEL)
+inline nxt_int_t
+nxt_otel_validate_endpoint(nxt_conf_validation_t *vldt,
+                           nxt_conf_value_t *value,
+                           void *data)
+{
+    // This function is a stub for now
+    return NXT_OK;
+}
+
+
+nxt_int_t
+nxt_otel_validate_batch_size(nxt_conf_validation_t *vldt,
+                             nxt_conf_value_t *value,
+                             void *data)
+{
+    double batch_size;
+    batch_size = nxt_conf_get_number(value);
+    if (batch_size <= 0) {
+      return NXT_ERROR;
+    }
+
+    return NXT_OK;
+}
+
+nxt_int_t
+nxt_otel_validate_sample_ratio(nxt_conf_validation_t *vldt,
+                               nxt_conf_value_t *value,
+                               void *data)
+{
+    double sample_ratio;
+
+    sample_ratio = nxt_conf_get_number(value);
+    if (sample_ratio < 0 || sample_ratio > 1) {
+        return NXT_ERROR;
+    }
+
+    return NXT_OK;
+}
+
+nxt_int_t
+nxt_otel_validate_protocol(nxt_conf_validation_t *vldt,
+                           nxt_conf_value_t *value,
+                           void *data)
+{
+    nxt_str_t proto;
+
+    nxt_conf_get_string(value, &proto);
+    if (nxt_str_eq(&proto, "HTTP", 4) ||
+        nxt_str_eq(&proto, "http", 4)) {
```

Comment:
In Unit we put the operator on the next line and when we have multi-line if's the `{` goes on it its own line.